.circleci/config.yml

```
@@ -792,6 +792,11 @@ jobs:
       - store_artifacts:
           path: development/ts-migration-dashboard/build
           destination: ts-migration-dashboard
+      - run:
+          name: Set branch parent commit env var
+          command: |
+            echo "export PARENT_COMMIT=$(git rev-parse origin/HEAD)" >> $BASH_ENV
```

This will produce misleading results whenever the branch is outdated. Like this PR for example! The MetaMask bot just said that the bundle size would increase, but this is not in fact true.

Instead perhaps we should include the actual parent commit of the first commit on the branch. Maybe something like

```suggestion
            echo "export PARENT_COMMIT=$(git merge-base --fork-point origin/HEAD HEAD)" >> $BASH_ENV
```